k8s.tf

```diff
@@ -104,7 +104,7 @@ resource "kubernetes_deployment_v1" "waggle_dance" {
           }
           liveness_probe {
             http_get {
-              path = "/actuator/health"
+              path = "/actuator/health/liveness"
```

Comment:
it looks like you also need to update https://github.com/ExpediaGroup/apiary-federation/blob/a68f7bbc7dabf804edaae65c7c87585e3f817c73/templates/waggledance.json#L56
And https://github.com/ExpediaGroup/apiary-federation/blob/a68f7bbc7dabf804edaae65c7c87585e3f817c73/ecs.tf#L115

Reply:
Changed